Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -972,52 +971,57 @@ static ErrorOr<NonnullOwnPtr<BitBuffer>> generic_region_decoding_procedure(Gener
     };
 
     // Figure 3(a) – Template when GBTEMPLATE = 0 and EXTTEMPLATE = 0,
-    constexpr auto compute_context_0 = [](NonnullOwnPtr<BitBuffer> const& buffer, int x, int y) -> u16 {
+    constexpr auto compute_context_0 = [](NonnullOwnPtr<BitBuffer> const& buffer, AdaptiveTemplatePixel const* adaptive_pixels, int x, int y) -> u16 {
```

Comment:
Let's use `ReadonlySpan<AdaptiveTemplatePixel>` instead of raw ptrs

Reply:
Sure, done. I was worried about perf since this is the innermost loop and it's "obviously" in bounds, but it doesn't have a measurable perf impact (and the current impl is offensively inefficient anyways). Thanks for making me measure 🙂